src/main/java/bio/terra/service/dataset/flight/ingest/IngestInsertIntoDatasetTableStep.java

```diff
@@ -49,6 +49,12 @@ public StepResult doStep(FlightContext context) throws InterruptedException {
       BulkLoadArrayResultModel fileLoadResults =
           workingMap.get(IngestMapKeys.BULK_LOAD_RESULT, BulkLoadArrayResultModel.class);
       ingestResponse.loadResult(fileLoadResults);
+      Long failedRowCount = 0L;
+      if (workingMap.containsKey(IngestMapKeys.COMBINED_FAILED_ROW_COUNT)) {
```

Comment:
It looks like this is needed due to the skippable step logic; it would be better IMO if a skippable step could always set its outputs so a downstream step didn't need to do this check, and know what the proper default value is.

Reply:
Not a bad idea!

Reply:
Actually, this check isn't needed because the `COMBINED_FAILED_ROW_COUNT` value will always exist in the context of the outer `if`